source/includes/kubernetes/_k8_statefulsets.md

```diff
@@ -0,0 +1,334 @@
+                }
+              ]
+            }
+          ],
+          "dnsPolicy": "ClusterFirst",
+          "restartPolicy": "Always",
+          "schedulerName": "default-scheduler",
+          "securityContext": {},
+          "terminationGracePeriodSeconds": 30,
+          "volumes": [
+            {
+              "configMap": {
+                "defaultMode": 420,
+                "items": [
+                  {
+                    "key": "aerospike.conf",
+                    "path": "aerospike.conf"
+                  }
+                ],
+                "name": "test-aerospike"
+              },
+              "name": "config-volume"
+            }
+          ]
+        }
+      },
+      "updateStrategy": {
+        "rollingUpdate": {
+          "partition": 0
+        },
+        "type": "RollingUpdate"
+      }
+    },
+    "status": {
+      "collisionCount": 0,
+      "currentReplicas": 1,
+      "currentRevision": "test-aerospike-6db7776c7d",
+      "observedGeneration": 1,
+      "readyReplicas": 1,
+      "replicas": 1,
+      "updateRevision": "test-aerospike-6db7776c7d",
+      "updatedReplicas": 1
+    }
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/statefulsets/:id</code>
+
+Retrieve a stateful set and all its info in a given [environment](#administration-environments).
+
+| Attributes                                 | &nbsp;                                                                |
+| ------------------------------------------ | --------------------------------------------------------------------- |
+| `id` <br/>*string*                         | The id of the stateful set                                            |
+| `metadata` <br/>*object*                   | The metadata of the the stateful set                                  |
+| `metadata.creationTimestamp` <br/>*string* | The date of creation of the stateful set as a string                  |
+| `metadata.labels` <br/>*map*               | The labels associated to the stateful set and there associated values |
+| `metadata.name` <br/>*string*              | The name of the stateful set                                          |
+| `metadata.namespace` <br/>*string*         | The namespace in which the stateful set is created                    |
+| `metadata.uid` <br/>*object*               | The UUID of the stateful set                                          |
+| `spec`<br/>*object*                        | The specification used to create and run the stateful set             |
+| `status`<br/>*object*                      | The status information of the the stateful set                        |
+| `status.readyReplicas`<br/>*integer*       | Total number of ready pods targeted by this deployment                |
+| `status.replicas`<br/>*integer*            | Total number of non-terminated pods targeted by this deployment       |
+
```

Comment:
Why is the same info repeated 4 times? Just curious.

Reply:
The docs needs to exist under the gcp plugin and the k8s standalone plugin. Then in each of those there's `/get` and /get/id`. The attributes are the same throughout.

Reply:
Thanks for clarifying!